### ISSUE TITLE
[suggestion] Push reply archives to archive.org before purging

### ISSUE THREAD

**Issue by Oflameo** [ZeroK-RTS/Zero-K-Infrastructure#2348]
I have a suggestion. To keep a longer backlog of replays, we could push our replays to archive.org before deleting them from the server. This is something I can actually write. I could use wget or curl, but I would probably use curl.

**Comment by sprunk** [comment]
I think they might already be pushed to glacier, check if that is the case.

**Comment by Licho1** [comment]
yes they are stored in glacier. Code that unfreezes them on demand could be added.
